omero/sysadmins/troubleshooting.txt

```diff
@@ -342,6 +344,33 @@ OMERO.web did not start
 
      :ome-users:`upgrade 5.0.5 to 5.1.1 omero.web forbidden <2015-April/005316.html>`
 
+OMERO.web running but status says not started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you upgraded OMERO but forgot to stop OMERO.web, processes will be still
+running. In order to kill stale processes by hand, run:
```

Comment:
You can use `run::` here and omit the following `::`.  Not needed to change here, just something to consider for future changes.